uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-calendar-legend.html

```diff
@@ -12,10 +12,25 @@ <h2 class="panel-title chr_leave-calendar__legend__title" role="button" ng-click
       <div class="chr_leave-calendar__legend__header">Types</div>
       <div class="clearfix">
         <div ng-repeat="absenceType in legend.absenceTypes"
-          class="chr_leave-calendar__legend__type-badge"
+          ng-click="legend.toggleFilteringByAbsenceType(absenceType.id)"
+          ng-class="{
+            'chr_leave-calendar__legend__type-badge-not-selected':
+              !legend.checkIfAbsenceTypeIsSelectedForFiltering(absenceType.id),
+            'chr_leave-calendar__legend__type-badge-selected':
+              legend.checkIfAbsenceTypeIsSelectedForFiltering(absenceType.id) && legend.absenceTypesToFilterBy.length}"
```

Comment:
why do we need this ` && legend.absenceTypesToFilterBy.length` when the absence type has been selected? I think this was already handled by `legend.checkIfAbsenceTypeIsSelectedForFiltering`

Reply:
`legend.checkIfAbsenceTypeIsSelectedForFiltering` will return TRUE if `legend.absenceTypesToFilterBy.length`, meaning, this Absence type will be **included** in the results. But in this particular case we need to apply class if ONLY it was selected manually.